admin/jqadm/themes/admin.js

```diff
@@ -21,6 +21,10 @@ Aimeos = {
 
 	editortags : 'div(*);span(*);p(*);',
 
+	extraPlugins: 'divarea',
+
+	removeButtons: 'Superscript,Subscript,JustifyBlock,Underline',
```

Comment:
Is this the default value?

Reply:
`editortags` is the default value that Aimeos defined on the Vue component definition directly; it is now moved to *admin.js`'s Aimoes config section, the Vue Component now defines `extraAllowedContent: Aimeos.editortags`; this makes it possible to be configurable by a user override.

Detto for `extraPlugins`.

`removeButtons`: this is unclear since I cannot find any information about what the "Standard all" configuration of CKEditor really disables (A question on Stackoverflow about exactly this topic is still unanswered). The current options "Superscript..." are the result of trial-and-error on my behalf, but maybe there are more buttons removed (like "table" options? "code"?)... It could also be set to an empty string, which would simply enable all buttons. Problematic topic... what do you recommend?

Reply:
@removeButtons: sorry..."Bretter vorm Kopf"... "console.log(CKEDITOR)" helps. So after commenting out each and every "Aimeos.removeButtons" config option, the consolge.log show:

`CKEDITOR.instances['cke-0'].config.removeButtons = "Underline,Subscript,Superscript"`

That seems to be the standard. I'll fix that.